src/fonts/ploticon/config.json

```diff
@@ -1513,4 +1513,4 @@
       "src": "fontawesome"
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
Noise. Please undo formatting changes like this one.

Reply:
@etpinard not me, sublime text fixes absence of new line for me. It is going to be tricky to look up for such bits

Reply:
Can you just commits file that modified on-purpose? This will make my life a lot easier. Thank you.